cdm_processing/cdm_processing_settings.py

```diff
@@ -18,7 +18,7 @@ class CdmProcessingSettings:
     output_path: str
     mapping_settings: MappingSettings
     profile: bool
-    label_sub_folder: Optional[str] = None
+    labels: Optional[str] = None
```

Comment:
Renamed since it can now can be either a path or a boolean indicating a subfolder